### DISCUSSION TITLE
Feature Request: Windows 10's News and interests

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1674] [Ideas]
Is it possible to backport the Weather current conditions from Windows 10 into ExplorerPatcher? And while on it - maybe we could also backport the Windows 10 News and Interests (which is smaller than 11's) into there as well?
![Annotation 2023-06-06 093201 (2)](https://github.com/valinet/ExplorerPatcher/assets/140480110/1ccdc339-57fa-4a5b-9b42-e1ce6aa48760)

**Comment by XuanChen1234** [comment]
This is a great restoration project but the old windows 10 News and interests would be SO much appreciated.

**Comment by XuanChen1234** [comment]
Any more planned?

**Comment by pyrates999** [comment]
This is also a no.

**Comment by XuanChen1234** [comment]
@Amrsatrio

**Comment by XuanChen1234** [comment]
Ask for the option to add News and interests.

**Reply by milestprower92** [reply]
Not possible
but the button can be added but it will open the widgets dashboard

**Reply by XuanChen1234** [reply]
Ask for the option to add Windows 10's News and interests!

**Comment by XuanChen1234** [comment]
@Vipin2003 Ask for the option to add Windows 10's News and interests!

**Comment by vippium** [comment]
Afterall, u are asking for Widgets on Taskbar10,., @Amrsatrio can give better answer.

**Reply by Amrsatrio** [reply]
No.

**Comment by XuanChen1234** [comment]
You can also use Windows 10's News and interests on Taskbar10, @Amrsatrio can give better answer.

**Comment by XuanChen1234** [comment]
@Vipin2003 Windows 10의 뉴스 및 관심사를 추가하는 옵션을 요청하십시오!

**Comment by pyrates999** [comment]
This should be closed as this feature request was declined.

**Comment by XuanChen1234** [comment]
@HAYDENCRAFT2008 Any plans for the Windows 10's News and interests flyout?
